src/WormholeJoin.sol

```diff
@@ -222,7 +224,7 @@ contract WormholeJoin {
         wormholes[hashGUID].blessed = true;
         wormholes[hashGUID].pending = wormholeGUID.amount;
         emit Register(hashGUID, wormholeGUID);
-        return _mint(wormholeGUID, hashGUID, maxFeePercentage, operatorFee);
+        (postFeeAmount, totalFee) = _mint(wormholeGUID, hashGUID, maxFeePercentage, operatorFee);
```

Comment:
it's the same thing isn't it?

Reply:
Yeah, that part it was just a cosmetic change.